content/_data/changelogs/lts.yml

```
@@ -9733,6 +9733,526 @@
+  - type: rfe
+    category: rfe
+    pull: 8262
+    issue: 71496
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-71496] Make tab pane selection reachable via keyboard"
+    message: |-
+      Make tab panes accessible via keyboard.
+  - type: rfe
+    category: rfe
+    authors:
+      - philfry
+    pr_title: "RPM: Remove System V initialization script"
+    references:
+      - url: https://github.com/jenkinsci/packaging/pull/409
+        title: pull 409 (packaging)
+      - url: https://www.jenkins.io/blog/2022/03/25/systemd-migration/
+        title: Linux install packages migrated from System V init to systemd
+    message: |-
+      Remove System V initialization scripts from RPM based installers.
+      The System V initialization scripts were replaced in March 2022 with systemd initialization.
+      RPM users with a custom log directory no longer have a <code>logrotate(8)</code> configuration out-of-the-box.
+  - type: rfe
+    category: rfe
+    pull: 8273
+    issue: 71366
+    authors:
+      - mattpulver
+    pr_title: "[JENKINS-71366] Set sandbox allow-same-origin in DEFAULT_CSP_VALUE."
+    message: |-
+      Add <code>allow-same-origin</code> to the <code>sandbox</code> ContentSecurityPolicy directive of workspace and artifact browsers if the Resource Root URL feature is not used.
+      Allow requests to resources like stylesheets and images, even if a reverse proxy prohibits cross-site requests.
+  - type: rfe
+    category: rfe
+    pull: 8129
+    authors:
+      - janfaracik
+    pr_title: Add last build status to job page
+    message: |-
+      Add last build status to job page.
+  - type: rfe
+    category: rfe
+    pull: 8258
+    authors:
+      - daniel-beck
+    pr_title: Remove Rebuilder Plugin from setup wizard options
+    message: |-
+      Remove the rebuild plugin from the setup wizard plugin selection.
+  - type: bug
+    category: bug
+    pull: 8425
+    issue: 71479
+    authors:
+      - dwnusbaum
+    pr_title: "[JENKINS-71479] Do not use SCSS lighten function directly to avoid\
+      \ invalid CSS"
+    message: |-
+      Fix invalid CSS which caused some buttons to become invisible on hover.
+  - type: bug
+    category: bug
+    pull: 8529
+    issue: 72067
+    authors:
+      - basil
+    pr_title: "[JENKINS-72067] High memory usage from `XStream2.AssociatedConverterImpl`"
+    message: |-
+      Reduce high memory usage from <code>XStream2.AssociatedConverterImpl</code>.
+  - type: bug
+    category: bug
+    pull: 8474
+    authors:
+      - jtnord
+    pr_title: fix signup autofocus
+    message: |-
+      Prevent log spam when using the Jenkins security database and users signup.
+  - type: bug
+    category: bug
+    pull: 8493
+    issue: 71880
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-71880] fix notification in breadcrumb"
+    message: |-
+          Show a confirmation popup when triggering <code>l:task</code> action from context menu.
```

I think that users aren't helped by the code in this case.
```suggestion
          Show a confirmation popup when triggering a task action from a context menu.
```